server/Arcadia.Assistant.Web/Infrastructure/WebTelemetryInitializer.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Arcadia.Assistant.Web.Infrastructure
+{
+    using Microsoft.ApplicationInsights.Channel;
+    using Microsoft.ApplicationInsights.Extensibility;
+
+    public class WebTelemetryInitializer : ITelemetryInitializer
+    {
+        public void Initialize(ITelemetry telemetry)
+        {
+            telemetry.Context.Cloud.RoleName = "Arcadia Assistant Web";
```

Comment:
Doesn't make sense to have it in settings?

Reply:
no